modules/installation-configuration-parameters.adoc

```diff
@@ -807,6 +810,10 @@ in vSphere.
 |====
 |Parameter|Description|Values
 
+|`platform.vsphere.clusterOSImage`
+|The location from which the installer downloads the {op-system} image. You must set this parameter to perform an installation in a restricted network.
+|An HTTP or HTTPS URL, optionally with a SHA-256 checksum. For example, `\https://mirror.openshift.com/images/rhcos-<version>-vmware.<architecture>.ova`.
+
 |`platform.vsphere.osDisk.diskSizeGB`
 |The size of the disk in gigabytes.
 |Integer.
```

Comment:
In the "Values" column for the tables in this module, there is a mix use of a period at the end of the value. For example, "String" vs "String."